index.html

```diff
@@ -0,0 +1,230 @@
+        <h2>About Myself</h2>
+        <p>
+            Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        
+        <h3 class="connect">LET'S CONNECT</h3>
+        
+        <ul class="icons">
+            <li>
+                <a href="https://twitter.com/malikhaiderkha"><img src="./assets/images/twitter.png"></a>
+            </li>
+
+            <li>
+                <a href="https://www.linkedin.com/in/malik-haider-khan-b53188140"><img src="./assets/images/linkedin.png"></a>
+            </li>
+
+            <li>
+                <a href="#"><img src="./assets/images/M.png"></a>
+            </li>
+
+            <li>
+                <a href="https://github.com/malikhaiderkhan"><img src="./assets/images/github.png"></a>
+            </li>
+
+            <li>
+                <a href="#"><img src="./assets/images/victory.png"></a>
+            </li>
+        </ul>
+
+        <input class="btn" id="resume" type="button" value="Get my resume">
+        </div>
+
+        <ul class="about2">
+            <li class="tskill">
+              <p>Languages</p>
+              <a href="#"><img src="./assets/images/down.png" alt="drop"></a>
+            </li>
+
+            <li>
+              <ul class="lang">
+                <li>
+                  <img src="./assets/images/js.png" alt="javascript" />
+                  <p>JavaScript</p>
+                </li>
+
+                <li>
+                  <img src="./assets/images/html.png" alt="html" />
+                  <p>HTML</p>
+                </li>
+                <li>
+                  <img src="./assets/images/css.png" alt="css" />
+                  <p>CSS</p>
+                </li>
+              </ul>
+            </li>
+            <li class="tskill">
+              <p>Frameworks</p>
+              <a href="#"><img src="./assets/images/right.png" alt="right"></a>
+            </li>
+            <hr />
+            <li class="tskill">
+              <p>Skills</p>
+              <a href="#"><img src="./assets/images/right.png" alt="right"></a>
+            </li>
+            <hr />
+          </ul>
+    </section>
+    <section class="contact-section">
+        <section class="contact">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                I’d love to help with it!</p>
+                <form
+                action="https://formspree.io/f/xpzebdqb"
+                method="POST" class="fcontact"
+              >
+            <input type="text" name="name" id="name" class="plc" placeholder="Lorem Ipsum" maxlength="30" required>
+            <input type="email" name="email" id="email" class="plc" pattern="[^ @]*@[^ @]*" placeholder="loremipsum@gmail.com"  required>
```

Comment:
- [ ] `[OPTIONAL]`  Since the primary purpose of a `placeholder` text is to tell users the kind of information they need to input in a provided input field. I kindly suggest you change your `placeholder` text to do just that as you did on the `textarea`.